test/Microsoft.NET.Build.Containers.IntegrationTests/CreateNewImageTests.cs

```diff
@@ -52,7 +52,7 @@ public void CreateNewImage_Baseline()
         task.BaseImageTag = "7.0";
 
         task.OutputRegistry = "localhost:5010";
-        task.LocalRegistry = DockerAvailableFactAttribute.LocalRegistry;
+        task.LocalRegistry = DockerAvailableFactAttribute.Attribute.LocalRegistry;
```

Comment:
Doesn't look like this change was intentional as it doesn't compile.

Reply:
Yep, so much for my easy ctrl f comment 😄